unlock-app/src/components/interface/LoginPrompt.tsx

```diff
@@ -108,6 +116,19 @@ const LoginPrompt = ({
     await authenticateIfNotHandled(ethereum)
   }
 
+  const tryAutoLogin = useCallback(() => {
```

Comment:
I wonder if that logic wouldn't be better off in a different location? I worry that this might create a flash?

Reply:
Checked also simulating a slow network connection and no flash effect. as you suggested I can move it to a different location like `DashboardContent`, and also include a loader to prevent any eventual flash

Reply:
Great idea and good news! 
I think it would be better in the `<Authenticate>` component as it is happening at the highest level and can set the values in the provider directly? Wdyt?